### PR TITLE
Feature/change order ownership

### PR DESCRIPTION
**Pull request by InnaBoitsun**

### CODE REVIEW

File: src/Orckestra.Composer.Website/UI.Package/LocalizedStrings/MyAccount.resx [hunk 1]
```diff
@@ -490,4 +490,7 @@ Don't worry, you can still find great products to offer!</value>
   <data name="B_EditOrder" xml:space="preserve">
     <value>Edit Order</value>
   </data>
+  <data name="B_ContinueEditing" xml:space="preserve">
+    <value>Continue Edititng</value>
```
Comment (vadym-hyryn) [inline src/Orckestra.Composer.Website/UI.Package/LocalizedStrings/MyAccount.resx:494]: Editi**t**ng

File: src/Orckestra.Composer.Cart/Factory/Order/OrderDetailsViewModelFactory.cs [hunk 2]
```diff
@@ -139,10 +139,26 @@ protected virtual OrderDetailInfoViewModel GetOrderInfosViewModel(CreateOrderDet
...
             return orderInfos;
         }
 
+        protected virtual bool HasOwnDraft(CreateOrderDetailViewModelParam param)
+        {
+            var orderDraft = param.OrderCartDrafts?.FirstOrDefault(d => Guid.Parse(d.Name) == Guid.Parse(param.Order.Id));
+            if (orderDraft != null)
+            {
+                orderDraft.PropertyBag.TryGetValue(Constants.OrderDart.OwnershipPropertyBagKey, out object orderDraftOwnershipUserName);
+                if (orderDraftOwnershipUserName != null && Constants.OrderDart.OwnershipByWebsite == orderDraftOwnershipUserName.ToString())
```
Comment (vadym-hyryn) [inline src/Orckestra.Composer.Cart/Factory/Order/OrderDetailsViewModelFactory.cs:153]: Can be simplified as 
```if (orderDraftOwnershipUserName?.ToString() == Constants.OrderDart.OwnershipByWebsite)```

**Review by vadym-hyryn — APPROVED** [review]

File: src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs [hunk 3]
```diff
@@ -452,25 +474,71 @@ public async Task<EditingOrderViewModel> CreateEditingOrderViewModel(string orde
...
+
+                    cartSummary = draftCarts?.FirstOrDefault(d => Guid.Parse(d.Name) == orderId);
+                    if (cartSummary == null)
+                    {
+                        throw new InvalidOperationException("Expected draft cart, but received null.");
+                    }
+
+                } else
+                {
+                    throw (ex);
```
Comment (vadym-hyryn) [inline src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs:519]: Better to use `throw;`

File: src/Orckestra.Composer.Cart/Factory/Order/OrderDetailsViewModelFactory.cs (hunk 2 above)
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Cart/Factory/Order/OrderDetailsViewModelFactory.cs]: fixed. Thanks

File: src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs (hunk 3 above)
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs]: fixed. Thanks

File: src/Orckestra.Composer.Website/UI.Package/LocalizedStrings/MyAccount.resx (hunk 1 above)
Comment (InnaBoitsun) [reply src/Orckestra.Composer.Website/UI.Package/LocalizedStrings/MyAccount.resx]: fixed. Thanks

**Pull request merged**